cypher.go

```diff
@@ -1,40 +1,22 @@
 package main
 
 import (
-	log "github.com/Sirupsen/logrus"
-	"github.com/jmcvetta/neoism"
+	"github.com/Financial-Times/neo-utils-go/neoutils"
 )
 
-// Driver interface
-type Driver interface {
-	ConnectedPeople(uuid string, fromDateEpoch int64, toDateEpoch int64, limit int, minimumConnections int, contentLimit int) (connectedPeople []ConnectedPerson, found bool, err error)
-	MostMentioned(fromDateEpoch int64, toDateEpoch int64, limit int) (thingList []Thing, found bool, err error)
+type driver interface {
+	ConnectedPeople(uuid string, fromDateEpoch int64, toDateEpoch int64, limit int, minimumConnections int, contentLimit int) ([]ConnectedPerson, bool, error)
```

Comment:
The `found` flag basically abstracts the fact that it's an empty result, and we can handle it more elegantly in the caller code rather than checking explicitly the length of the returned slice.